### PR TITLE
Separate db create from schema create

### PR DESCRIPTION
**Pull request by niksajakovljevic**
There are some cases when database needs to be configured/setup in a certain way (eg. setup db using some other tooling). So preferably you'd only want to drop the metrics table and create a new one and leave the rest of db intact.

### CODE REVIEW

File: cmd/tsbs_load_timescaledb/creator.go [hunk 1]
```diff
@@ -76,75 +76,80 @@ func (d *dbCreator) CreateDB(dbName string) error {
 	db := sqlx.MustConnect(dbType, d.connStr)
 	db.MustExec("CREATE DATABASE " + dbName)
 	db.Close()
+	return nil
+}
 
-	dbBench := sqlx.MustConnect(dbType, getConnectString())
-	defer dbBench.Close()
+func (d *dbCreator) PostCreateDB(dbName string) error {
+	if createFactsTable {
```
Comment (RobAtticus) [inline cmd/tsbs_load_timescaledb/creator.go]: `createMetricsTable` maybe?

File: cmd/tsbs_load_timescaledb/creator.go [hunk 2]
```diff
@@ -76,75 +76,80 @@ func (d *dbCreator) CreateDB(dbName string) error {
...
+		createTagsTable(dbBench, parts[1:])
+		tableCols[tagsKey] = parts[1:]
+
+		for _, cols := range d.cols {
+			parts = strings.Split(strings.TrimSpace(cols), ",")
+			hypertable := parts[0]
+			partitioningField := tableCols[tagsKey][0]
+			tableCols[hypertable] = parts[1:]
+
+			psuedoCols := []string{}
```
Comment (RobAtticus) [inline cmd/tsbs_load_timescaledb/creator.go]: can you fix my spelling mistake? :) `pseudoCols`

**Comment by RobAtticus** [comment]
Also, please add an extended commit description of what the purpose of this change is/why.

**Comment by niksajakovljevic** [comment]
@RobAtticus Sure, I'll add commit description

**Pull request merged**